Phoenix/PHShebangPreprocessor.m

```diff
@@ -67,7 +67,11 @@ + (NSString *) process:(NSString *)script atPath:(NSString *)path error:(NSError
     NSPipe *standardError = [NSPipe pipe];
     NSFileHandle *standardOutputFile = standardOutput.fileHandleForReading;
 
-    task.launchPath = @"/bin/bash";
+    // Make sure we use the current user's $SHELL
+    NSDictionary *environmentDict = [[NSProcessInfo processInfo] environment];
+    NSString *shellString = [environmentDict objectForKey:@"SHELL"];
+
+    task.launchPath = shellString;
     task.standardOutput = standardOutput;
     task.standardError = standardError;
     task.arguments = @[ @"-cl", [NSString stringWithFormat:@"%@ %@", command, path] ];
```

Comment:
Looks fairly safe to use the `"-cl"` options for _any_ shell http://hyperpolyglot.org/unix-shells#toc83